src/plugins/LADTable/components/LADRow.vue

```diff
@@ -89,43 +94,80 @@ export default {
             .telemetry
             .subscribe(this.domainObject, this.updateValues);
 
-        this.openmct
-            .telemetry
-            .request(this.domainObject, {strategy: 'latest'})
-            .then((array) => this.updateValues(array[array.length - 1]));
+        this.requestHistory();
     },
     destroyed() {
         this.stopWatchingMutation();
         this.unsubscribe();
-        this.openmct.off('timeSystem', this.updateTimeSystem);
+        this.openmct.time.off('timeSystem', this.updateTimeSystem);
+        this.openmct.time.off('bounds', this.updateBounds);
     },
     methods: {
         updateValues(datum) {
-            this.timestamp = this.formats[this.timestampKey].format(datum);
-            this.value = this.formats[this.valueKey].format(datum);
-
-            var limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+            let newTimestamp = this.formats[this.timestampKey].parse(datum),
+                shouldUpdate = false,
```

Comment:
I think all the nested ifs below can be eliminated by doing this
`shouldUpdate = this.timestamp === '---' || newTimestamp >= this.timestamp`
followed by 
```
if(!this.inBounds(newTimestamp))
return;
```
and then all you need is
```
if (shouldUpdate) {
....do your thing here.
}
```

Reply:
I agree! Those nested ifs are very... nested. If I did this, I'd still need to check if the existing is in bounds before comparison with the new one though, BUT I think I can do this if I reset the existing timestamp on bounds change back to '---' then everything should proceed as normal, so I'll reset the timestamp on bounds change and use this new implementation you suggested which is much more clear and succinct.